### PR TITLE
Home/SignUp/LogIn react

### PR DESCRIPTION
**Pull request by EmilianoC02**
Created new directory in src folder called home. This directory contains Landing.js, SignUp.js, and LogIn.js files. Created react components for those routes

### CODE REVIEW

File: student-hub-app/src/Home/LogIn.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+            <h2>Login</h2>
+            <form onSubmit={handleSubmit}>
+                <label>
+                    Email:
+                    <input
+                        type="email"
+                        name="email"
+                        value={formData.email}
+                        onChange={handleChange}
+                    />
```
Comment (ryanmercado) [inline student-hub-app/src/Home/LogIn.js:37]: Are we logging in with email rather than username?

File: student-hub-app/src/Home/SignUp.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+                        name="email"
+                        value={formData.email}
+                        onChange={handleChange}
+                    />
+                </label>
+                <br />
+                <label>
+                    Password:
+                    <input
+                        type="password"
```
Comment (ryanmercado) [inline student-hub-app/src/Home/SignUp.js:55]: We should put a confirm password box here too

**Review by ryanmercado — APPROVED** [review]
We should make my commented changes, but let's still approve so I can add backend functionality

File: student-hub-app/src/Home/Landing.js
Comment (cdewitt02) [inline student-hub-app/src/Home/Landing.js:1]: Are we going to do a landing page or a login page with also a signup when the user first opens the app?

**Review by cdewitt02 — APPROVED** [review]
A few things to consider (see comments) but overall ready to merge to main in my opinion

File: student-hub-app/src/Home/Landing.js
Comment (Grayson34) [reply student-hub-app/src/Home/Landing.js:1]: login with also signup. If they choose sign up they will be redirected to a signup page where they enter their info. After clicking enter on the signup page they are redirected to the login page.

File: student-hub-app/src/Home/LogIn.js (hunk 1 above)
Comment (Grayson34) [reply student-hub-app/src/Home/LogIn.js:37]: ^^. This is signup but still, should also have username. Maybe email, username, and password

File: student-hub-app/src/Home/SignUp.js (hunk 2 above)
Comment (Grayson34) [reply student-hub-app/src/Home/SignUp.js:55]: ^

File: student-hub-app/src/Home/SignUp.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+const SignUp = () => {
+    const [formData, setFormData] = useState({
+        username: '',
+        email: '',
+        password: '',
+    });
+
+    const handleChange = (e) => {
+        setFormData({ ...formData, [e.target.name]: e.target.value });
```
Comment (Grayson34) [inline student-hub-app/src/Home/SignUp.js:13]: what is the ...?

File: student-hub-app/src/Home/Landing.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+                <button>Login</button>
+            </Link>
+            <Link to="/signup">
+                <button>Sign Up</button>
+            </Link>
+        </div>
+    );
+};
+
+export default Landing;
```
Comment (Grayson34) [inline student-hub-app/src/Home/Landing.js:20]: login and landing page should be the same. dont think we need this one

**Pull request merged**